packages/app/src/cli/services/deploy/mode.ts

```diff
@@ -25,6 +28,9 @@ export async function resolveDeploymentMode(app: OrganizationApp, options: Deplo
     }
   }
 
+  await metadata.addPublicMetadata(() => ({
+    cmd_app_deployment_mode: deploymentMode,
```

Comment:
Could we set `cmd_app_deployment_mode` =  'unified' when deploymentMode === 'unified-skip-release'. 'unified-skip-release' means that is using `unified` mode and the flag `--no-relase` but maybe it could be better to track `--no-release` with a new separate monorail attribute